cli/cmd/charmil/locales/en/add.en.yaml

```diff
@@ -0,0 +1,25 @@
+add.cmd.use:
```

Comment:
This is outside the scope of this PR, but should we localize the use of command?

Reply:
i am not sure about this. It's being done in rhoas as well.